api/src/routes/team.js

```diff
@@ -74,4 +74,20 @@ teamRouter.patch(
   teamController.updateTeam,
 );
 
+teamRouter.delete(
+  '/:team_id',
+  authMiddleware.cookieJwtAuth,
+  mongooseMiddleware.validateTeamObjectId,
+  teamMiddleware.validateTeamId,
+  teamController.deleteTeam
+);
+
+teamRouter.delete(
+  '/:team_id/deleteGHPages/:code',
+  authMiddleware.cookieJwtAuth,
+  mongooseMiddleware.validateTeamObjectId,
+  teamMiddleware.validateTeamId,
+  teamController.deleteGHPages
```

Comment:
Also, the backend functionality is missing from this PR?

Reply:
wait what?